docs/fleet/machines.md

```diff
@@ -25,63 +25,56 @@ Click the name of a machine to go to that machine's page, where you'll find a va
 
 ## Navigating the machine page
 
-The banner at the top of the machine page displays the machine's location, name, and a dropdown list of all {{< glossary_tooltip term_id="part" text="parts" >}} of that machine.
+Click on the **parts live** display at the top left corner of the page to open a menu with information about each {{< glossary_tooltip term_id="part" text="part" >}} of this machine.
```

Comment:
I wasn't quite sure what a parts live display is, maybe status indicator makes more sense?
```suggestion
Next to the machine name, there is an indicator of the machine's status.
Click on the **status** dropdown to open a menu with information about each {{< glossary_tooltip term_id="part" text="part" >}} of your machine.
```
Also thre's no _this_, it's a random machine they'll be seeing.

Reply:
What I meant is that it says "0/2 parts live" or similar when not live. Definitely better to consider both views